.gitignore

```diff
@@ -1,6 +1,7 @@
 all_data
 SentenceText-2022-05-01.csv
 wiki-full-data.csv
+ai4bharat-indic-nlp-f0df53305a6f.json
```

Comment:
What if we change the keys in the future? :)

I guess `*.json` is OK.
Similarly `*.csv` above

Reply:
If we really to need add a JSON or CSV file, we can force add by `git add -f <filename>`

Reply:
Okay, makes sense. I will remove this json file and an exception for all of them. I didn't add these csv files to the `.gitignore` but I can remove them too and add an exception for them.